frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -34,9 +36,26 @@ const menuActions = [
   ...common,
 ];
 
-const PVCStatus = ({ pvc }) => (
-  <Status status={pvc.metadata.deletionTimestamp ? 'Terminating' : pvc.status.phase} />
-);
+const PVCStatus = ({ pvc }) => {
+  const pvcStatusExtensions = useExtensions(isPVCStatus);
+  if (pvcStatusExtensions.length > 0) {
+    const priorityStatus = pvcStatusExtensions.reduce((prev, current) =>
+      prev.properties.priority > current.properties.priority ? prev : current,
+    );
+    const predicate = priorityStatus.properties.predicate;
+
+    const priorityStatusComponent = predicate(pvc) && (
```

Comment:
you need to pick the extension status which returns true and has highest priority. 

Doing it this way, if the extension status with highest priority returns false, you will just ignore the others and fallback to default status which is not correct.